server/routes.js

```diff
@@ -15,13 +16,15 @@ module.exports = function(app) {
   app.use('/api/things', require('./api/thing'));
   app.use('/api/users', require('./api/user'));
 
+  app.use('/uploads', express.static( __dirname + "/static/uploads" ));
```

Comment:
Could we move this outside of the project directory? the current deployment system relies on the ability to move the entire project structure but photos shouldn't be reset between versions.